.changeset/cold-schools-yell.md

```diff
@@ -0,0 +1,34 @@
+---
+'astro': minor
+---
+
+Add integrations dynamically
```

Comment:
I'm torn, because while I think this is a good blog header, I think in our changesets starting with "add", typically we'd be saying what we added, which is the ability to add integrations dynamically. :sweat_smile: 

> Add the ability to add integrations dynamically

Reply:
Ah yeah, definitely was thinking of the blog format here and you’re right it’s a strange fit for the changeset.

This might work? (Also avoids the “add … add”.)

```suggestion
Support adding integrations dynamically
```

Reply:
I'll let Mattthew be the final judge here on what's helpful for him! But sure, "support adding", "support for adding"... those work!